.gitlab/internal_kubernetes_deploy.yml

```diff
@@ -51,7 +55,6 @@ notify-slack:
     - if: $DDR != "true"
       when: never
     - !reference [ .on_deploy_a7 ]
-  image: registry.ddbuild.io/slack-notifier:sdm
   tags: ["arch:amd64"]
   needs: ["internal_kubernetes_deploy_experimental"]
   script:
```

Comment:
Wouldn't keeping the `script` section prevent the template from doing what it needs to do?

Reply:
Yes we do the same for `notify` job. The idea is to use the `.slack-notifier-base` to have the correct image and we can override the script and do what we want

Reply:
ah indeed! Thanks for the clarification!